maintenance_plan_activity/tests/test_maintenance_plan_activity.py

```diff
@@ -49,11 +49,11 @@ def test_01_cron_auto_create_activities(self):
         # Check if activity Call has been created for the request 1
         generated_activities = self.mail_activity_obj.search(
             [('res_id', '=', request_1.id)])
-        self.assertEqual(len(generated_activities), 1)
+        self.assertEqual(len(generated_activities), 2)
         self.assertEqual(
             generated_activities[0].activity_type_id.name, self.call.name)
         self.assertEqual(
-            generated_activities[0].date_deadline,
+            str(generated_activities[0].date_deadline),
```

Comment:
It is simpler to keep this as it was and delete the following `fields.Date.to_string`

Reply:
okei